src/PowerShellEditorServices/Services/TextDocument/Handlers/PsesSemanticTokensHandler.cs

```diff
@@ -25,7 +25,7 @@ protected override SemanticTokensRegistrationOptions CreateRegistrationOptions(S
             {
                 Delta = true
             },
-            Range = true
```

Comment:
@david-driscoll do you have any idea why the server is failing to initialize with that error? The type of `Range` here is `BooleanOr<SemanticTokensCapabilityRequestRange>` so it should be fine but I'm getting a serialization exception.